public/video-ui/src/components/VideoUpload/AddYouTubeAsset.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import Icon from '../Icon';
+import { ManagedForm, ManagedField } from '../ManagedForm';
+import SelectBox from '../FormFields/SelectBox';
+import { channelAllowed } from '../../util/channelAllowed';
+
+export default class PlutoProjectPicker extends React.Component {
+  state = { file: null };
+
+  setFile = event => {
+    if (!this.props.video) {
+      return;
+    }
+
+    if (event.target.files.length == 0) {
+      this.setState({ file: null });
+    } else {
+      this.setState({ file: event.target.files[0] });
+    }
+  };
+
+  render() {
+    const { video, channels, startUpload } = this.props;
+    const hasAssets = video.assets.length !== 0;
+
+    const isHosted = video.category === 'Hosted';
+    const isManaged = channelAllowed(video, channels);
+
+    const editable =
+      !hasAssets && (!video.channelId || !!video.youtubeCategoryId);
```

Comment:
I think the check here should be checking for youtube assets rather than all assets.